public/src/components/channelManagement/bannerDesigns/BannerDesignForm.tsx

```diff
@@ -106,6 +98,7 @@ const BannerDesignForm: React.FC<Props> = ({
     const isValid = Object.keys(errors).length === 0;
     onValidationChange(isValid);
   }, [
+    // TODO - can we just use errors here?
```

Comment:
Not sure if you meant to include this TODO? (Fine either way!)

Reply:
removed - will address in another PR